thirdparty/build_gstreamer.sh

```diff
@@ -1,9 +1,10 @@
 cd gst-build
+rm -rf gst-build-1.16 || true
```

Comment:
The cache restores this, so why should we delete it ? I guess leaing this folder as-is will make rebuild fatser ? what am I missing ?

Reply:
so this was a temporary solution , to a strange behavior I was noticing , which was build script getting triggered even though the cache was hit and then failing due to already present cache. Also the cache which is present was obsolete since it did not include json-glib. We have to somehow invalidate the previous cache or give our cache keys more context if its still not invalidated.